app/routes.py

```diff
@@ -95,3 +95,41 @@ def get_id(kml_admin_id):
         ),
         200
     )
+
+
+@app.route('/kml/<kml_admin_id>', methods=['DELETE'])
+def delete_id(kml_admin_id):
+    enforcer = DynamoDBFilesHandler(
+        table_name=AWS_DB_TABLE_NAME,
+        bucket_name=AWS_S3_BUCKET_NAME,
+        table_region=AWS_DB_REGION_NAME,
+        endpoint_url=AWS_DB_ENDPOINT_URL
+    )
+    item = enforcer.get_item(kml_admin_id)
+
+    # Fetching a non existing Item will return "None"
+    if item is None:
+        logger.error("Could not find the following kml id in the database: %s", kml_admin_id)
+        abort(400, f"Could not find {kml_admin_id} within the database.")
```

Comment:
Here we should return a 404 instead of 400 for resource that are not found (not existing)